Src/WitsmlExplorer.Frontend/components/ContentViews/QueryView.tsx

```diff
@@ -67,6 +67,7 @@ const QueryView = (): React.ReactElement => {
   const [isLoading, setIsLoading] = useState(false);
   const [isXmlResponse, setIsXmlResponse] = useState(false);
   const [returnElements, setReturnElements] = useState(ReturnElements.All);
+  const [optionsIn, setOptionsIn] = useState<string>("");
```

Comment:
True. IMO, both are fine as writing it explicitly makes it more readable. This might not make a big difference for strings, but it makes a bigger difference for example in the line above for ReturnElements.